### PR TITLE
Add Sterling Bills Payment to Nodejs Module

### PR DESCRIPTION
**Pull request by Akinpelumi**

### CODE REVIEW

File: lib/sterling/BillPayment.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            method: 'GET',
+            path: '/sterling/billpaymentapi/api/Spay/GetBillersISWRequest',
+            credentials: { ...data },
+        });
+    } catch (error) {
+        return error;
+    }
+};
+
+exports.BillersISW = async(data) => {
```
Comment (mezlet) [inline lib/sterling/BillPayment.js:28]: BillerPaymentItems and BillersISW are fetching from the same endpoint.

**Review by mezlet — CHANGES_REQUESTED** [review]

File: lib/sterling/BillPayment.js (hunk 1 above)
Comment (Akinpelumi) [reply lib/sterling/BillPayment.js:28]: yes true... based on the documentation link i showed you, that was what I saw

Comment (mezlet) [reply lib/sterling/BillPayment.js:28]: That is an error. Check the endpoints, they are different

Comment (Akinpelumi) [reply lib/sterling/BillPayment.js:28]: the difference i saw from that link i sent you was in the query parameter.... the one for BillerPaymentItems has billerID

**Review by mezlet — APPROVED** [review]

**Review by akinjide — APPROVED** [review]

**Pull request merged**